app/views/layouts/_nav.html.erb

```diff
@@ -1,5 +1,55 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-  <div class="container">
-    <%= link_to 'Nav', fake_index_path, class: 'navbar-brand' %>
+<nav class="prx-dovetail-nav navbar navbar-expand-sm navbar-dark m-0 p-0">
+  <div class="container-fluid pe-0">
+    <%= link_to 'Dovetail', fake_index_path, class: 'navbar-brand' %>
+    <ul class="navbar-nav d-flex flex-row justify-content-end align-items-center">
+      <li class="nav-item">
+        <a class="nav-link active" href="/">Podcasts</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="https://augury.prx.org">Inventory</a>
+      </li>
+      <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+          <span class="material-icons align-middle">help</span>
+        </a>
+        <ul class="dropdown-menu dropdown-menu-end shadow" aria-labelledby="navbarDropdown">
+          <li><a class="dropdown-item" href="https://help.prx.org/">Help Docs</a></li>
+          <li><a class="dropdown-item" href="http://status.prx.org/">Status</a></li>
+          <li><a class="dropdown-item" href="https://exchange.prx.org/terms-of-use">Terms</a></li>
+          <li><a class="dropdown-item" href="https://dovetail.prx.org/attribution">Attribution</a></li>
+          <li><a class="dropdown-item" href="https://exchange.prx.org/privacy-policy">Privacy</a></li>
+          <li><a class="dropdown-item" href="https://prx.org">PRX</a></li>
+        </ul>
+      </li>
+      <% if current_user %>
```

Comment:
Yeah, doesn't exist yet.  Will work on that soon.

We may also want to split this file out a bit at some point.  Getting a bit large.

Reply:
+1 Maybe pull the account stuff into its own layout. For another day :)